bootstrap/kubeadm/config/manager/manager.yaml

```diff
@@ -20,14 +20,17 @@ spec:
         - /manager
         args:
         - "--leader-elect"
-        - "--metrics-bind-addr=127.0.0.1:8080"
+        - "--metrics-bind-addr=:8080"
```

Comment:
Just checked and found out that this is the default value. So we don't have to set it. I honestly don't particularly care if we remove the argument altogether or explicitly set the default. Maybe others do

Reply:
Note: to me it looks like the section title `Required kustomize changes to have a single manager watching all namespaces and answer to webhook calls` does not really accurately describe all things we change in the kustomize folder. So it might make sense to make the section title a bit more generic.

Reply:
If this is the default I'm +1 to remove this flag if possible